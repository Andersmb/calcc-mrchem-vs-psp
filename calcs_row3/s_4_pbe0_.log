Starting job 5688142 on c2-[30-31,47-48],c10-5 at Fri May 20 09:16:32 CEST 2022

Copying s_4_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/singlets
Copying s_4_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/singlets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688142       s_4_pbe0_        180                                             00:30:40      0:0 
5688142.bat+      batch         36        1   00:00:02          0   00:00:02   00:30:40      0:0 
5688142.ext+     extern        180        5   00:00:00          3   00:00:00   00:30:40      0:0 
5688142.0      mrchem.x        180       30   00:30:20         29   01:27:25   00:30:25      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688142                                                                          
5688142.bat+     19280K          0     19280K        0              0          0 
5688142.ext+          0          3          0        0              3          0 
5688142.0     11687892K         29 4690180642        0             15          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688142                                                                                               
5688142.bat+       16.90M               0         16.90M        0.03M                0          0.03M 
5688142.ext+        0.00M               3          0.00M            0                3              0 
5688142.0           7.32M              24          6.85M      121.94M                0         63.18M 

Job 5688142 completed at Fri May 20 09:47:13 CEST 2022
