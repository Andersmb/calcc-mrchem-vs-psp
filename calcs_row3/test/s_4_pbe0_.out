                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       S            :       -0.053996        0.000000        1.943016
    1       S            :       -0.053996        0.000000       -1.943016
    2       H            :        1.717811        1.781067        2.290726
    3       H            :        1.717811       -1.781067       -2.290726
---------------------------------------------------------------------------
 Center of mass          :       -0.000000       -0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              17
 Beta electrons          :                                              17
 Total electrons         :                                              34
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              9.999997029060e-01
    1    1    a          :                              9.999985648948e-01
    2    1    a          :                              1.000001146263e+00
    3    1    a          :                              1.000000085419e+00
    4    1    a          :                              1.000000792263e+00
    5    1    a          :                              1.000000069107e+00
    6    1    a          :                              9.999996720042e-01
    7    1    a          :                              9.999999317052e-01
    8    1    a          :                              9.999999900645e-01
    9    1    a          :                              1.000000018552e+00
   10    1    a          :                              9.999998744320e-01
   11    1    a          :                              9.999999018038e-01
   12    1    a          :                              1.000000499208e+00
   13    1    a          :                              1.000000163530e+00
   14    1    a          :                              9.999998594018e-01
   15    1    a          :                              1.000000010929e+00
   16    1    a          :                              1.000000029609e+00
   17    1    b          :                              9.999996793610e-01
   18    1    b          :                              9.999968878193e-01
   19    1    b          :                              1.000000042501e+00
   20    1    b          :                              1.000000026231e+00
   21    1    b          :                              1.000000437100e+00
   22    1    b          :                              1.000000019652e+00
   23    1    b          :                              9.999982940769e-01
   24    1    b          :                              9.999998496719e-01
   25    1    b          :                              9.999998263975e-01
   26    1    b          :                              1.000001168841e+00
   27    1    b          :                              9.999998738657e-01
   28    1    b          :                              9.999998674578e-01
   29    1    b          :                              1.000000957172e+00
   30    1    b          :                              1.000000147069e+00
   31    1    b          :                              9.999999448434e-01
   32    1    b          :                              1.000000011387e+00
   33    1    b          :                              1.000000027421e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                791.744278185744
 E-N energy              :            (au)              -2058.763003347749
 Coulomb energy          :            (au)                440.362294935639
 Exchange energy         :            (au)                -12.678878726072
 X-C energy              :            (au)                -39.114430342139
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 85.166568080644
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -878.449739294578
 Nuclear energy          :            (au)                 85.166568080644
---------------------------------------------------------------------------
 Total energy            :            (au)             -7.932831712139e+02
                         :      (kcal/mol)             -4.977927015351e+05
                         :        (kJ/mol)             -2.082764664575e+06
                         :            (eV)             -2.158633382559e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -88.379681122452
    1    1    a          :            (au)                -88.376293425669
    2    1    a          :            (au)                 -7.933532071364
    3    1    a          :            (au)                 -7.929783307073
    4    1    a          :            (au)                 -5.876685052120
    5    1    a          :            (au)                 -5.872782706826
    6    1    a          :            (au)                 -5.870099407922
    7    1    a          :            (au)                 -5.867405123552
    8    1    a          :            (au)                 -5.861593114348
    9    1    a          :            (au)                 -5.858054886062
   10    1    a          :            (au)                 -0.848155211191
   11    1    a          :            (au)                 -0.695825507956
   12    1    a          :            (au)                 -0.473589322241
   13    1    a          :            (au)                 -0.404404412169
   14    1    a          :            (au)                 -0.343389056373
   15    1    a          :            (au)                 -0.261216219142
   16    1    a          :            (au)                 -0.258816468733
   17    1    b          :            (au)                -88.379810615151
   18    1    b          :            (au)                -88.376179322166
   19    1    b          :            (au)                 -7.933580632702
   20    1    b          :            (au)                 -7.929731150100
   21    1    b          :            (au)                 -5.876561005962
   22    1    b          :            (au)                 -5.872905740041
   23    1    b          :            (au)                 -5.870116110347
   24    1    b          :            (au)                 -5.867402193259
   25    1    b          :            (au)                 -5.861594602085
   26    1    b          :            (au)                 -5.858065800684
   27    1    b          :            (au)                 -0.848171432616
   28    1    b          :            (au)                 -0.695813776021
   29    1    b          :            (au)                 -0.473589503107
   30    1    b          :            (au)                 -0.404397133403
   31    1    b          :            (au)                 -0.343388442048
   32    1    b          :            (au)                 -0.261222361229
   33    1    b          :            (au)                 -0.258813263531
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -462.222649499643
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          5.830952e+00        -793.283171213934         -7.932832e+02
    1          2.533052e-01        -797.204290276149         -3.921119e+00
    2          8.386522e-02        -797.328359907551         -1.240696e-01
    3          4.198073e-02        -797.348686115319         -2.032621e-02
    4          5.401641e-02        -797.356860587860         -8.174473e-03
    5          2.852515e-02        -797.358684961049         -1.824373e-03
    6          1.803673e-02        -797.359460410443         -7.754494e-04
    7          6.508651e-03        -797.359594348554         -1.339381e-04
    8          5.012074e-03        -797.359633578095         -3.922954e-05
    9          2.228720e-03        -797.359645425513         -1.184742e-05
   10          1.994412e-03        -797.359654560057         -9.134544e-06
   11          1.012270e-03        -797.359656505730         -1.945673e-06
   12          5.252879e-04        -797.359657124707         -6.189766e-07
   13          3.705761e-04        -797.359657288769         -1.640624e-07
   14          2.910614e-04        -797.359657417995         -1.292259e-07
   15          2.452249e-04        -797.359657479147         -6.115249e-08
   16          1.066242e-04        -797.359657494672         -1.552405e-08
---------------------------------------------------------------------------
                      SCF converged in 16 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       S            :       -0.053996        0.000000        1.943016
    1       S            :       -0.053996        0.000000       -1.943016
    2       H            :        1.717811        1.781067        2.290726
    3       H            :        1.717811       -1.781067       -2.290726
---------------------------------------------------------------------------
 Center of mass          :       -0.000000       -0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                796.246820213156
 E-N energy              :            (au)              -2066.980152936718
 Coulomb energy          :            (au)                440.038729731268
 Exchange energy         :            (au)                -12.682206373458
 X-C energy              :            (au)                -39.149416209563
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 85.166568080644
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -882.526225575315
 Nuclear energy          :            (au)                 85.166568080644
---------------------------------------------------------------------------
 Total energy            :            (au)             -7.973596574947e+02
                         :      (kcal/mol)             -5.003507352765e+05
                         :        (kJ/mol)             -2.093467477756e+06
                         :            (eV)             -2.169726066342e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -89.053816647138
    1    1    a          :            (au)                -89.053812362338
    2    1    a          :            (au)                 -8.032713366628
    3    1    a          :            (au)                 -8.032625806777
    4    1    a          :            (au)                 -5.970079925129
    5    1    a          :            (au)                 -5.969818903676
    6    1    a          :            (au)                 -5.966172434202
    7    1    a          :            (au)                 -5.966122675336
    8    1    a          :            (au)                 -5.960624138668
    9    1    a          :            (au)                 -5.960621215469
   10    1    a          :            (au)                 -0.830673851203
   11    1    a          :            (au)                 -0.690335256339
   12    1    a          :            (au)                 -0.470653099736
   13    1    a          :            (au)                 -0.424791072103
   14    1    a          :            (au)                 -0.374718165182
   15    1    a          :            (au)                 -0.279925039359
   16    1    a          :            (au)                 -0.278659557037
   17    1    b          :            (au)                -89.053816540464
   18    1    b          :            (au)                -89.053812466551
   19    1    b          :            (au)                 -8.032713315887
   20    1    b          :            (au)                 -8.032625857709
   21    1    b          :            (au)                 -5.970079927624
   22    1    b          :            (au)                 -5.969818903320
   23    1    b          :            (au)                 -5.966172451222
   24    1    b          :            (au)                 -5.966122655773
   25    1    b          :            (au)                 -5.960624138948
   26    1    b          :            (au)                 -5.960621214195
   27    1    b          :            (au)                 -0.830673846269
   28    1    b          :            (au)                 -0.690335251287
   29    1    b          :            (au)                 -0.470653100488
   30    1    b          :            (au)                 -0.424791065656
   31    1    b          :            (au)                 -0.374718158143
   32    1    b          :            (au)                 -0.279925036435
   33    1    b          :            (au)                 -0.278659558767
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -466.632327005060
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -1.256905       -0.000017        0.000037
 Magnitude               :            (au)                        1.256905
                         :         (Debye)                        3.194733
---------------------------------------------------------------------------
 Nuclear vector          :        1.707735        0.000000        0.000000
 Magnitude               :            (au)                        1.707735
                         :         (Debye)                        4.340629
---------------------------------------------------------------------------
 Total vector            :        0.450830       -0.000017        0.000037
 Magnitude               :            (au)                        0.450830
                         :         (Debye)                        1.145896
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 30m 21s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
